jupyter/datascience/ubi9-python-3.11/runtime-images/pytorch-ubi9-py311.json

```diff
@@ -5,7 +5,7 @@
       "pytorch"
     ],
     "display_name": "Pytorch with CUDA and Python 3.11 (UBI9)",
-    "image_name": "quay.io/opendatahub/workbench-images@sha256:6a2806bf3cd9b00f60f3c7fe907727fb954c9776a075d9d58df26b5119d7afe6",
+    "image_name": "quay.io/opendatahub/workbench-images@",
```

Comment:
looks like this wasn't updated properly

Reply:
yeah... the quay.io wasn't respond.I will update that in a separate commit

Reply:
fixed that